tests/main/presenters/test_search_results.py

```diff
@@ -47,6 +47,12 @@ def setup_method(self, method):
             self._get_framework_fixture_data('g-cloud-6')['frameworks']
         )
 
+    def test_search_results_accepts_top_level_documents_key_in_response_instead_of_services(self):
+        self.fixture['documents'] = self.fixture.pop('services')
+
+        search_results_instance = SearchResults(self.fixture, self._lots_by_slug)
+        assert hasattr(search_results_instance, 'search_results')
```

Comment:
I don't think this test can fail - if you call the constructor, it will always have that attribute. You need to test for `None`, I think?

Reply:
You can do `getattr(search_results_instance, "search_results", None) is not None` to catch either case.

Reply:
I was testing here that you can instantiate a SearchResults object with a response that has a `documents` key rather than a `service` key. If you change the key to anything else the test will fail.

Reply:
Maybe I could just change to `assert SearchResults(self.fixture, self._lots_by_slug)`

Reply:
Have updated!